share/instruments/instruments.xml

```diff
@@ -7279,6 +7279,125 @@
+                        <stem>1</stem>
+                  </Drum>
+                  <Drum pitch="41"> <!--Low Floor Tom-->
+                        <head>normal</head>
+                        <line>5</line>
+                        <voice>0</voice>
+                        <name>Floor Tom</name>
+                        <stem>1</stem>
+                  </Drum>
+                  <Drum pitch="44"> <!--Pedal Hi-hat-->
+                        <head>cross</head>
+                        <line>9</line>
+                        <voice>2</voice>
+                        <name>Pedal Hi-Hat</name>
+                        <stem>2</stem>
+                        <shortcut>F</shortcut>
+                  </Drum>
+                  <Drum pitch="42"> <!--Closed Hi-hat-->
+                        <head>cross</head>
+                        <line>-1</line>
+                        <voice>0</voice>
+                        <name>Closed Hi-Hat</name>
+                        <stem>1</stem>
+                        <shortcut>G</shortcut>
+                  </Drum>
+                  <Drum pitch="46"> <!--Open Hi-hat-->
+                        <head>cross</head>
+                        <line>-1</line>
+                        <voice>0</voice>
+                        <name>Open Hi-Hat</name>
+                        <stem>1</stem>
+                  </Drum>
+                  <Drum pitch="51"> <!--Ride Cymbal 1-->
+                        <head>cross</head>
+                        <line>0</line>
+                        <voice>0</voice>
+                        <name>Ride Cymbal</name>
+                        <stem>1</stem>
+                        <shortcut>D</shortcut>
+                  </Drum>
+                  <Drum pitch="53"> <!--Ride Bell-->
+                        <head>cross</head>
+                        <line>0</line>
+                        <voice>0</voice>
+                        <name>Ride Bell</name>
+                        <stem>1</stem>
+                  </Drum>
+                  <Drum pitch="49"> <!--Crash Cymbal 1-->
+                        <head>cross</head>
+                        <line>-2</line>
+                        <voice>0</voice>
+                        <name>Crash Cymbal</name>
+                        <stem>1</stem>
+                        <shortcut>C</shortcut>
+                  </Drum>
+                  <Drum pitch="52"> <!--Chinese Cymbal-->
+                        <head>cross</head>
+                        <line>-3</line>
+                        <voice>0</voice>
+                        <name>China Cymbal</name>
+                        <stem>1</stem>
+                  </Drum>
+                  <Drum pitch="55"> <!--Splash Cymbal-->
+                        <head>cross</head>
+                        <line>-4</line>
+                        <voice>0</voice>
+                        <name>Splash Cymbal</name>
+                        <stem>1</stem>
+                  </Drum>
```

Comment:
We should see a `<Channel>` section after this. The reason we don't is you got the ID wrong in the [Channels sheet](https://docs.google.com/spreadsheets/d/1SwqZb8lq5rfv5regPSA10drWjUAoi65EuMoYtG-4k5s/edit#gid=504647632). You wrote `drum-kit` instead of `drum-kit-5`.